src/java/main/org/apache/zookeeper/Login.java

```diff
@@ -45,6 +47,7 @@
 import java.util.Set;
 
 public class Login {
+    private static final String KINIT_COMMAND_DEFAULT = "/usr/bin/kinit";
```

Comment:
1) No, KINIT_COMMAND_DEFAULT holds the kinit command to be used when kinit command is not configured. So it is really a default value and can not be replaced with KINIT_COMMAND

2) KINIT_COMMAND_DEFAULT is used only in Login.java, so i think it is OK to keep it here only.